### PR TITLE
Add additional productivity tips

### PR DESCRIPTION
**Pull request by adfoster-r7**
Adding two new useful productivity tips to the msfconsole

## Verification

- [ ] Start `msfconsole`
- [ ] `tip -l`
- [ ] **Verify** the tips are available

### CODE REVIEW

**Comment by h00die** [comment]
Is it reopen, or open? I think it opens the last open session.
I think the word open got confusing, opening a session vs a n open (aka valid and connected) session

**Comment by adfoster-r7** [comment]
@h00die Thanks - I've reworded it now, let me know if that's clearer 🤞

**Comment by gwillcox-r7** [comment]
Sorry @h00die, your welcome to land this if you want to, didn't meant to take over this issue. As for changes everything looks good, my only concern would be would doing `session -i 3` count as "opening" as session or just interacting with it? I'm thinking its interacting with it given what I've read, but just wanted to remove any confusion on that front. Otherwise looks good to merge.

**Comment by adfoster-r7** [comment]
@gwillcox-r7 There might be a race condition here. I had put up an initial tip that used the word `reopen`, and it's now been changed to `interact` :+1:

File: lib/msf/ui/tip.rb [hunk 1]
```diff
@@ -33,6 +33,8 @@ def self.highlight(string)
         "Metasploit can be configured at startup, see #{highlight('msfconsole --help')} to learn more",
         "Display the Framework log using the #{highlight('log')} command, learn more with #{highlight('help log')}",
         "Adapter names can be used for IP params #{highlight('set LHOST eth0')}",
+        "Use #{highlight('sessions -1')} to interact with the last opened session",
```
Comment (adfoster-r7) [inline lib/msf/ui/tip.rb:36]: Details on the implementation behind this one - when passing only a number to the session, it defaults to interact mode:

https://github.com/rapid7/metasploit-framework/blob/2a6a8e4c23aa57cbda5a93ec9229fd9ac43112e6/lib/msf/ui/console/command_dispatcher/core.rb#L1150-L1153

The session manager is then hardcoded to support the -1 value to get the highest session id:

https://github.com/rapid7/metasploit-framework/blob/2a6a8e4c23aa57cbda5a93ec9229fd9ac43112e6/lib/msf/core/session_manager.rb#L277-L280

Comment (gwillcox-r7) [reply lib/msf/ui/tip.rb:36]: Looks good to me, thanks for clarifying this @adfoster-r7, I'll go ahead and merge this so long.

**Comment by gwillcox-r7** [comment]
# Release Notes
This PR adds in two additional productivity tips to the tip command that help users be more efficient.

**Pull request merged**